Standards/scs-XXXX-v1-default-rules-for-security-groups.md

```diff
@@ -0,0 +1,116 @@
+| ID                       | IP Protocol | Ethertype | IP Range  | Port Range | Direction | Remote Security Group | Remote Address Group | Used in default Security Group | Used in custom Security Group |
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+| 47b929fd-9b39-4f22-abc5- | None        | IPv6      | ::/0      |            | egress    | None                  | None                 | True                           | True                          |
+| 7d4f64d10909             |             |           |           |            |           |                       |                      |                                |                               |
+| 92a79600-5358-4fef-a390- | None        | IPv4      | 0.0.0.0/0 |            | egress    | None                  | None                 | True                           | True                          |
+| 822665f46070             |             |           |           |            |           |                       |                      |                                |                               |
+| 93e35d0c-2482-4ec1-9fbd- | None        | IPv4      | 0.0.0.0/0 |            | ingress   | PARENT                | None                 | True                           | False                         |
+| fd8c9a21a04e             |             |           |           |            |           |                       |                      |                                |                               |
+| ed5cd662-add2-4e42-b0a7- | None        | IPv6      | ::/0      |            | ingress   | PARENT                | None                 | True                           | False                         |
+| 3b585d348820             |             |           |           |            |           |                       |                      |                                |                               |
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+```
+
+## Related Documents
+
+The spec for introducing configurability for the default Security Groups Rules can be found [here](https://specs.openstack.org/openstack/neutron-specs/specs/2023.2/configurable-default-sg-rules.html).
+
+More about Security Groups as a resource in OpenStack can be found [here](https://docs.openstack.org/nova/latest/user/security-groups.html).
+
+## Conformance Tests
+
+The conformance tests should check for the absence of any ingress traffic rules in the `openstack security group rule list`.
+As having egress rules is allowed by this standard, but not forced and can be set in various ways, there will not be any tests for such rules.
```

Comment:
The standard explicitly says that egress SHOULD be allowed for both v4 and v6, so that should IMO be tested.

Reply:
It's the default setting in Openstack. I'm not sure what we are gaining by testing it.

Reply:
> It's the default setting in Openstack. I'm not sure what we are gaining by testing it.

Does the SCS mandate somewhere that CSPs may not deviate from OpenStack default settings unless SCS standards specify it?
If not, then I think the proposal is completely valid that we check a CSP has not changed this specific setting from its default (which would in turn violate the standard).

Reply:
@markus-hentsch is exactly on point, @artificial-intelligence. If it's written in the standard, there should be a test for it, no matter whether it's a default or not.

Otherwise you can have a non-compliant cloud (because someone changed the defaults) and the compliance checker still gives the green check mark.

Reply:
I have tests for the presence of all OpenStack-default default SG rules.
Test for rules:
- ingress rules, that only allow traffic from the same SG
- egress rules, that allow all kind of egress traffic for all SGs in both IPv4 and IPv6